applications/nrf5340_audio/tools/buildprog/buildprog.py

```diff
@@ -109,6 +110,16 @@ def __build_cmd_get(core: Core, device: AudioDevice, build: BuildType, pristine,
         if options.nrf21540:
             device_flag += " -DSHIELD=nrf21540ek_fwd"
 
+        if options.user_bt_name:
+            MAX_USER_NAME_LEN = 248
+            user_specific_bt_name = ("AUDIO_DEV_" + getpass.getuser())[:MAX_USER_NAME_LEN].upper()
```

Comment:
Pitfall: If the max name len is 248, then you need to deduct "AUDIO_DEV_" (10) from `:MAX_USER_NAME_LEN`

Reply:
I dont understand this. In my head, it is the user_specific_bt_name that has to have a max length of 248. You might not have noticed that the [:MAX_USER_NAME_LEN] is applied to ("AUDIO_DEV_" + getpass.getuser()). Otherwise, I dont understand what you mean.